### ISSUE TITLE
Autocompletion dropdown doesn't adjust while I type (Windows)

### ISSUE THREAD

**Issue by theSimplex** [autocomplete-python/autocomplete-python#257]
I have Python executable configured to face my python.exe file.
And it does appear when I start typing, but it doesn't adjust the completion while I type it just stays idle.

![atom](https://cloud.githubusercontent.com/assets/1194412/21240925/033d1dec-c2d3-11e6-80eb-92a129d5d5d0.gif)

**Comment by sadovnychyi** [comment]
What do you mean by "it doesn't adjust the completion while I type it"?

**Comment by theSimplex** [comment]
I mean it not filters out suggestions. Like if I have a variable name and even when I almost fully typed it in it will stay on top of the original list of suggestions (starting from abs and so forth) containing like hundred of options. Which hardly seems helpful.

**Comment by sadovnychyi** [comment]
Ok, got it. Can you see same behavior with other autocompletion providers? Like with built-in snippets provider. This seems very strange and I cannot suggest you anything.

**Comment by theSimplex** [comment]
No, it only happens for this plugin. I'll check if this is a Windows-related issue, later when I check atom on a Unix machine.

**Comment by sadovnychyi** [comment]
Try to disable/enable fuzzy matcher in plugin settings.

**Comment by theSimplex** [comment]
I've tried switching on and off almost all settings in the config page, unfortunately, behavior remains the same.

**Comment by theSimplex** [comment]
Works fine on ubuntu.

**Comment by icearith** [comment]
a stupid question, what does the left 'p' in the suggestion list means?

**Comment by sadovnychyi** [comment]
@icearith Function parameter/argument. https://github.com/atom/autocomplete-plus/pull/334

**Comment by agraubert** [comment]
I've started noticing this too.  The first thing that came to mind is that autocomplete-python appears to be presenting completions too early.  Autocomplete-plus has a setting that prevents completions from appearing until you've typed at least _n_ characters (presumably so that the language-specific provider can narrow down the list of suggestions first).  Is it possible that autocomplete-python is jumping the gun, and presenting the entire list of completions?

**Comment by krayzpipes** [comment]
I am having this same issue with Ubuntu Mate 18. Autocomplete-python begins to autocomplete but does not filter down to what I am typing, although I can find my desired object if I scroll down in the suggestions popup.

**Comment by krayzpipes** [comment]
Just figured it out.  In ide-python, I had the wrong path to my python environment. Therefore, ide-python and pyls would not work because it could not find the python-language-server.

After correcting the path to my python executable in the ide-python configuration, the auto completion works as expected.
